### DISCUSSION TITLE
Why do not capitalized the global variable?

### DISCUSSION THREAD

**Discussion by you-n-g** [serious-scaffold/ss-python#806] [Q&A] [answered]
Why do not capitalized the global variable?
I mean change global_settings  to GLOBAL_SETTINGS
https://github.com/serious-scaffold/ss-python/blob/c4acbb9b3de287d216a3a2930be73cb68d5db281/src/ss_python/settings.py#L34

**Comment by huxuan** [comment, marked as answer]
For the standalone settings module, we just follow the convention in FastAPI here [1].
If you mean the difference between `settings` and `global_settings`, actually I did not think about it but maybe it is OK to follow the same pattern?

[1] https://fastapi.tiangolo.com/advanced/settings/#settings-in-another-module

**Reply by you-n-g** [reply]
Thanks for responding quickly. This approach is a bit different from what I'm used to, and I'm just a bit curious about it. I think it's fine now.
